### ISSUE TITLE
Toggle detached mode

### ISSUE THREAD

**Issue by devhyunjae** [kamranahmedse/pennywise#77]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
when i press cmd + shift + d it changes to detached ode
to change to normal mode. have to click the icon on the doc

**Expected behavior**

want `cmd + shift + d` to be toggled detached mode <-> normal mode

**URL attempting to access**

**Screenshots (Optional But May Be Requested)**

**Pennywise Version:**

**OS / OS Version:** MAC

Any issue that doesn't have all of these filled in will be closed, if yours is closed reopen with everything filled in.

**Comment by The-Code-Monkey** [comment]
What penny-wise version

**Comment by devhyunjae** [comment]
@The-Code-Monkey 0.6.1

**Comment by The-Code-Monkey** [comment]
Please upgrade to 0.6.2 to see if the issue persists

**Comment by kamranahmedse** [comment]
@The-Code-Monkey it's missing functionality. Upgrading the version won't do anything :)
@devhyunjae thanks for opening the issue, will see what we can do.

**Comment by The-Code-Monkey** [comment]
Ok that's fine but I think if the version is out of date the first thing to be done is update and see if the error is still there as we may eventually update electron meaning it could be an electron issue why it didn't work on 1 version but does on another

**Comment by geigel** [comment]
Sometimes when my PW window is detached I want to be able to click and move the window quickly. Could you do something where if you hold down a key combination and are hovering over a detached PW window it would allow you to have focus on PW window for a moment as if it were attached? And then releasing that key combo would let it go back to detach state?

Otherwise agree that a toggling of detach state via global key combo would be nice.

**Comment by squatto** [comment]
Related to this issue (I can make a separate one if you'd prefer) is that the deactivation of detached mode whenever PW gets focus could maybe be tweaked to instead require manual deactivation/"reattachment". This is best illustrated with this situation that I just went through:

1. Start dev tutorial video
2. Detach PW
3. Move focus to IDE
4. Video ends
5. Move focus to PW, click "Next Video"
6. Move focus back to IDE
7. Forget that PW isn't detached still
8. Move focus back to PW
9. Detach PW
10. Move focus back to IDE

Admittedly, my actions are partly caused by the fact that I've been using [Fluid Browser](https://itunes.apple.com/us/app/fluid-browser/id1077036385?mt=12) previously, which has that behavior. Despite that, I do think it would be nice to keep PW detached until I attach it. Or possibly have a setting along the lines of "Cancel Detached Mode on Focus".

**Comment by skplunkerin** [comment]
I agree with @squatto , I would expect PW to stay detached until I re-attach it
